### DISCUSSION TITLE
All my Lora's disappeared on Swarm.

### DISCUSSION THREAD

**Discussion by rhd2330** [Stability-AI/StableSwarmUI#457] [General]
All my Lora's disappeared on Swarm.

All my Lora's disappeared, and the dropdown settings below it had disappeared. I don't know what I did; I was messing around with the depth settings, and everything disappeared. I have tried to get it back, but nothing I have done is working. This is the error it is showing me:

Error: Invalid value '' for parameter 'depth', must be type 'Int32'

I have not gotten it fixed yet, but if you would like to try to do what I think I did, I wouldn't advise it, but maybe you want to try. Everything else works fine; it's just everything under the Lora's disappears, and I mean everything.

In the depth field, type 1.1; that is what I think I did, and everything on Swarm vanished under Loras.

You may say, "Just type in a different value," but the place where you type in a different value is gone. There is nothing there. Everything else is working. If there is another location to get to it, please let me know. Thank you.

**Comment by jwax33** [comment]
Can you screenshot your Swarm interface?  I am unable to duplicate the issue by typing 1.1 into the Depth field.  Have you tried using the Quick Tools dropdown in the upper right and selecting"Reset Params to Default" to see if that fixes the value for you?

**Reply by rhd2330** [reply]
I have a friend on YouTube I contacted, and he did the 1.1 and got the same results, all of his Lora's disappear just like mine, here is how he fixed it, he went into his browser and deleted his cash, and somehow that fixed it, but that is not working for me, everything below the line of choices starting with image history, is completely black, all the other settings are working perfectly, I cannot figure it out, and it happened suddenly when I put in 1.1, all my Lora's just vanished.

I appreciate your help.

**Reply by rhd2330** [reply]
In connection to quick tools, the answer is yes, more times than I care to count.

**Reply by Jelosus2** [reply]
This is the old repository tho, your running an outdated version. Here's the link for the up-to-date version of SwarmUI: https://github.com/mcmonkeyprojects/SwarmUI

**Reply by rhd2330** [reply]
How do you install this update? Thank you for your reply

**Reply by rhd2330** [reply]
Is this the right version? SwarmUI v0.9.2.2 (2024-09-14 12:45:28)

Still do not have access to my Lora's on Swarm.

**Reply by rhd2330** [reply]
This is the error I keep getting, if I can get past this or reset this somehow, I do believe it will fix the issue, but I don't have access to that particular setting.

Error

Invalid value '1.1' for parameter 'depth', must be type 'Int32'

**Reply by mcmonkey4eva** [reply]
You're in the wrong place, you want https://github.com/mcmonkeyprojects/SwarmUI or the Discord https://discord.gg/q2y38cqjNw
